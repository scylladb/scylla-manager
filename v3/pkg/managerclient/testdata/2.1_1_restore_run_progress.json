{
  "progress": {
    "views": [
      {
        "completed_at": "2025-01-01T09:30:00.000Z",
        "keyspace": "keyspace1",
        "started_at": "2025-01-01T09:00:00.000Z",
        "status": "done",
        "table": "table1",
        "view": "view1",
        "view_type": "MaterializedView"
      },
      {
        "completed_at": "2025-01-01T09:30:00.000Z",
        "keyspace": "keyspace1",
        "started_at": "2025-01-01T09:00:00.000Z",
        "status": "done",
        "table": "table1",
        "view": "index1",
        "view_type": "SecondaryIndex"
      }
    ]
  },
  "run": {
    "end_time": "2025-01-01T10:00:00.000Z",
    "id": "task_run_id",
    "start_time": "2025-01-01T09:00:00.000Z",
    "status": "DONE"
  },
  "Detailed": true
}
