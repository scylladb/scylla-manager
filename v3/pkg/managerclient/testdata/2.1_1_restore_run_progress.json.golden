Run:		task_run_id
Status:		DONE
Start time:	01 Jan 25 10:00:00 CET
End time:	01 Jan 25 11:00:00 CET
Duration:	1h0m0s
Progress: 
╭───────────┬────────┬──────────────────┬──────────┬────────╮
│ Keyspace  │ View   │ Type             │ Duration │ Status │
├───────────┼────────┼──────────────────┼──────────┼────────┤
│ keyspace1 │ view1  │ MaterializedView │ 30m0s    │ done   │
│ keyspace1 │ index1 │ SecondaryIndex   │ 30m0s    │ done   │
╰───────────┴────────┴──────────────────┴──────────┴────────╯
